hirola/front/static/front/css/landing_page.css

```diff
@@ -1,2 +1,59 @@
-html, body {height: 100%; margin:0; padding:0;     flex: 1;
-  overflow-y: scroll;}
\ No newline at end of file
+html,
+body {
+    /* height: 100%; */
+    /* margin: 0;
+    padding: 0; */
+    /* flex: 1; */
+    /* overflow-y: scroll; */
+    /* height: 100vh; */
+}
+
+.carousel {
+    height: auto;
+    height: 93.5vh;
+}
+.content {
+    margin-bottom: 0 !important;
+}
+
+.item-one {
+    position: relative;
+    background-image: url('../../../static/front/phone8.jpeg');
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+    -moz-background-size: cover;
+}
+
+.item-two {
+    position: relative;
+    background-image: url('../../../static/front/phone7.jpeg');
```

Comment:
Also the css practically is repeating a lot of attributes. If you look at item one, two and three, they repeat the same thing except for the url. If the attributes were too different we would end up using an Id. So we'd have an id for item-one, two and three and that would make sense, but what is being used is a class. this means that we can have a class called `item-slideshow` and each image can inherit it on the html tag.

Reply:
I think in this case we should focus on inheritance.

Reply:
got It, that's a great idea